.* causes an "Invalid specifier" error with python3.11
--- setup.py.orig	2024-05-02 10:19:03.562524530 +0200
+++ setup.py	2024-05-02 10:19:16.950455438 +0200
@@ -43,7 +43,7 @@
         "Programming Language :: Python :: Implementation :: PyPy",
         "Programming Language :: Python :: Implementation :: CPython",
     ],
-    python_requires=">3.5.*",
+    python_requires=">3.5",
     package_data={"aiormq": ["py.typed"]},
     extras_require={
         "develop": [
